Services/AssessmentQuestion/interfaces/interface.ilAsqQuestionAuthoring.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/* Copyright (c) 1998-2013 ILIAS open source, Extended GPL, see docs/LICENSE */
+
+/**
+ * Interface ilAsqQuestionAuthoring
+ *
+ * @author    Björn Heyser <info@bjoernheyser.de>
+ * @version    $Id$
+ *
+ * @package    Services/AssessmentQuestion
+ */
+interface ilAsqQuestionAuthoring
+{
+	/**
+	 * Execute Command
+	 */
+	public function executeCommand();
+	
+	/**
+	 * @param ilAsqQuestion $question
+	 * @return void
+	 */
+	public function setQuestion(ilAsqQuestion $question);
+
+	/**
+	 * @param ilQuestionChangeListener $listener
+	 */
+	public function addQuestionChangeListener(ilQuestionChangeListener $listener);
+	
+	/**
+	 * @param object $a_object
+	 * @param string $a_method
+	 * @param mixed $a_parameters
+	 */
+	public function addNewIdListener($a_object, $a_method, $a_parameters = "");
+	
+	/**
+	 * @param int $a_new_question_id
+	 */
+	public function callNewIdListeners($a_new_question_id);
```

Comment:
I do not understand what this is about, a description of the php-doc would be great.

Reply:
A current requirement for the LM integration I cannot get rid of for the moment.